integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/createTable/TestNonTransactionalCarbonTable.scala

```diff
@@ -761,9 +847,29 @@ class TestNonTransactionalCarbonTable extends QueryTest with BeforeAndAfterAll {
     sql("select * from sdkOutputTable").show(false)
 
     checkAnswer(sql("select * from sdkOutputTable"), Seq(
-      Row("bob", "10", Row("abc","bang")),
-      Row("bob", "10", Row("abc","bang")),
-      Row("bob", "10", Row("abc","bang"))))
+      Row("bob", 10, Row("abc","bang")),
+      Row("bob", 10, Row("abc","bang")),
+      Row("bob", 10, Row("abc","bang"))))
+
+    sql("DROP TABLE sdkOutputTable")
+    // drop table should not delete the files
+    assert(new File(writerPath).exists())
```

Comment:
check for file existence inside writerPath instead of the folder

Reply:
done